helm/slurm-cluster/templates/slurm-cluster-cr.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
 spec:
   crVersion: {{ .Chart.Version }}
   pause: {{ .Values.pause }}
+  clusterType: {{ .Values.clusterType }}
```

Comment:
Maybe we can just use the number of "nvidia.com/gpu" instead of introducing a separate field?

Reply:
I'm asking because we'll definitely add support for separate "NodeSets", where nodes in each set may have different resources. So we'll have to drop this field.

Reply:
I think number of "nvidia.com/gpu" is very complicated logic... and "nvidia.com/gpu" number can be valid only for Nebius... Lets drop that field, with NodeSets?

Reply:
OK